homeworks/szabolcsgesztesi/4_exceptions_logging/.gitignore

```diff
@@ -0,0 +1 @@
+Task_Manager_logs.log
```

Comment:
Nem írja hogy empty file, itt van a Task_Manager_logs.log
Viszont a feladat megfogalmazása egy kicsit becsapós volt :) A .gitignore már alapból jelen van a repository root-jában:
https://github.com/Global-rd/python_hu/blob/main/.gitignore

Ebben a file-ban már ott van ez a sor:
https://github.com/Global-rd/python_hu/blob/880967aeb5b44c7c5e3b1a2c72c19177bebdf4ac/.gitignore#L21

a *.log-ban a * egy ún. wildcard, ami azt jelenti hogy minden file-t ami .log kiterjesztésű, ki fog zárni a verziókezelésből. Beleértve ebbe az almappákban lévő log file-okat is. Így neked külön nem kell egy .gitginore a mappádba, ezt töröld kérlek.